packages/augur-ui/src/modules/auth/constants/trezor-manifest.ts

```diff
@@ -0,0 +1,2 @@
+export const TREZOR_MANIFEST_EMAIL = "team@augur.net";
+export const TREZOR_MANIFEST_APP = "https://try.augur.net";
```

Comment:
what's the manifest app, what about mainnet?

Reply:
Trezor requires it, see below.

>> Starting with Trezor Connect 7, we have implemented a new feature — Trezor Connect Manifest — which requires that you as a Trezor Connect integrator, to share your e-mail and application url with us. This provides us with the ability to reach you in case of any required maintenance. This subscription is mandatory. Trezor Connect raises an error that reads "Manifest not set.